tools/lib/test_script.py

```diff
@@ -79,15 +80,31 @@ def get_provisioning_status() -> Tuple[bool, Optional[str]]:
     return False, preamble(version) + NEED_TO_UPGRADE
 
 
-def assert_provisioning_status_ok(force: bool) -> None:
-    if not force:
+def assert_provisioning_status_ok(skip_provision_check: bool) -> None:
+    if not skip_provision_check:
         ok, msg = get_provisioning_status()
         if not ok:
             print(msg)
-            print("If you really know what you are doing, use --force to run anyway.")
+            print(
+                "If you really know what you are doing, use --skip-provision-check to run anyway."
+            )
             sys.exit(1)
 
 
+def add_provision_check_override_param(parser: ArgumentParser, help: Optional[str] = None) -> None:
+    """
+    Registers --skip-provision-check argument to be used with various commands/tests in our tools.
+    """
+    if help is not None:
+        parser.add_argument("--skip-provision-check", action="store_true", help=help)
+    else:
+        parser.add_argument(
+            "--skip-provision-check",
+            action="store_true",
+            help="Skip check that provision has been run; useful when bisecting if you know the dependency changes are not relevant to your test",
```

Comment:
"Skip check that provision has been run; useful to save time if you know the dependency changes are not relevant to this command and will not cause it to fail." 

would be good enough to not need the if/else block.